<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(200, 200, 200);
background-color: rgb(10, 10, 10);R</string>
  </property>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55);
}
QLineEdit:focus {
	border: 2px solid rgb(255, 207, 0);	
	color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="placeholderText">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QPushButton" name="register_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>240</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(255, 144, 80);
	border: 2px solid rgb(255, 165, 24);	
	color: rgb(35, 35, 0);
}</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}
QLineEdit:hover {
	border: 2px solid rgb(55, 55, 55);
}
QLineEdit:focus {
	border: 2px solid rgb(255, 207, 0);	
	color: rgb(200, 200, 200);
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>密码</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="login">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {	
	background-color: rgb(50, 50, 50);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(255, 144, 80);
	border: 2px solid rgb(255, 165, 24);	
	color: rgb(35, 35, 0);
}</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="result" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>290</width>
     <height>110</height>
    </rect>
   </property>
   <widget class="QLabel" name="result_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>注册结果</string>
    </property>
   </widget>
   <widget class="QLabel" name="result_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>账号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
